test/build/config/type/function-with-env/webpack.env.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = (env) => {
+  console.log(env);
```

Comment:
PS: Is this expected 👀

Reply:
Yes, to check env values in test cases on stdout.